README.md

```diff
@@ -251,40 +251,31 @@ Only the release should merge into the main branch!!!
-[![@OlyaKorchan](https://avatars.githubusercontent.com/u/17857767?v=4)](https://github.com/OlyaKorchan)
-[![@Marichka0406](https://avatars.githubusercontent.com/u/121502737?v=4)](https://github.com/Marichka0406)
-
-### DevOps team
-
-[![@abohatyrov](https://avatars.githubusercontent.com/u/52012169?v=4)](https://github.com/abohatyrov)
-[![@bdeputat](https://avatars.githubusercontent.com/u/36072762?v=4)](https://github.com/bdeputat)
-
-### Designer team
-
-[![@Nastia197](https://avatars.githubusercontent.com/u/76164279?v=4)](https://github.com/Nastia197)
-
-### BA team
-
-[![@IvannaSW](https://avatars.githubusercontent.com/u/24367409?v=4)](https://github.com/IvannaSW)
-
-### QC team
-
-[![@AntonOkun](https://avatars.githubusercontent.com/u/129941062?v=4)](https://github.com/AntonOkun)
-[![@Valent1n0o](https://avatars.githubusercontent.com/u/118978192?v=4)](https://github.com/Valent1n0o)
-
----
+[![@ohanovdmytro](https://avatars.githubusercontent.com/ohanovdmytro?s=130)](https://github.com/ohanovdmytro)
+[![@Olenka-Hryk](https://avatars.githubusercontent.com/Olenka-Hryk?s=130)](https://github.com/Olenka-Hryk)
+[![@Nazar-Lysak](https://avatars.githubusercontent.com/Nazar-Lysak?s=130)](https://github.com/Nazar-Lysak)
+[![@docia](https://avatars.githubusercontent.com/docia?s=130)](https://github.com/docia)
+[![@BodiaPutiak](https://avatars.githubusercontent.com/BodiaPutiak?s=130)](https://github.com/BodiaPutiak)
+<a href="https://github.com/m6xcom"><img src="https://avatars.githubusercontent.com/m6xcom" height="auto" width="130"></a>
 
 ## FAQ
 
 - **How do I do _specifically_ so and so?**
   - No problem! Just do this.
 
----
+## 🔨 Tools
+
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/html5/html5-original.svg" alt="HTML5" align="left" width="40" height="60"/>  
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/css3/css3-original.svg" alt="CSS3" align="left" width="40" height="60"/>   
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/materialui/materialui-original.svg" alt="MaterialUI" align="left" width="40" height="60"/> 
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/javascript/javascript-original.svg" alt="JavaScript" align="left" width="40" height="60"/>   
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/react/react-original.svg" alt="React" align="left" width="40" height="60"/>  
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/redux/redux-original.svg" alt="Redux" align="left" width="40" height="60"/>
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/nodejs/nodejs-original.svg" alt="NodeJS" align="left" width="40" height="60"/>
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/mongodb/mongodb-original.svg" alt="MongoDB" align="left" width="40" height="60"/>   
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/eslint/eslint-original.svg" alt="Eslint" align="left" width="40" height="60"/> 
+<br />
+<br />
```

Comment:
```suggestion
```

Reply:
Without <br /> content, the bottom content moves up to the top.